contracts/modules/licensing/PILTermsRenderer.sol

```diff
@@ -21,6 +21,12 @@ contract PILTermsRenderer {
         // **Attributions**
         string memory json = string(
             abi.encodePacked(
+                '{"trait_type": "Royalty Policy", "value": "',
+                terms.royaltyPolicy.toHexString(),
+                '"},',
+                '{"trait_type": "Default Minting Fee", "value": "',
+                terms.defaultMintingFee.toString(),
+                '"},',
```

Comment:
Besides the two attribute, seems `transferable` is missing as well. could you also add the missing attribute?